openpype/hosts/maya/plugins/publish/extract_playblast.py

```diff
@@ -25,6 +36,31 @@ class ExtractPlayblast(publish.Extractor):
     optional = True
     capture_preset = {}
 
+    def _capture(self, preset, override_viewport_options, instance):
+        filename = preset.get("filename", "%TEMP%")
+
+        # Force viewer to False in call to capture because we have our own
+        # viewer opening call to allow a signal to trigger between
+        # playblast and viewer
+        preset['viewer'] = False
```

Comment:
I feel like we could also move this outside of the function. There's actually no reason why this was originally in the maintained time thing.

Reply:
Yeah, reducing it to just the debug and capture now.